client/my-sites/posts/style.scss

```diff
@@ -619,6 +623,7 @@
 	flex-grow: 1;
 	margin: 0 0 24px 24px;
 	display: none;
+	max-width: 400px;
```

Comment:
On bigger windows we don't want to artificially limit this at that size. Specially with images it becomes very narrow.